docs/reference/objects/product/extra.md

```diff
@@ -176,6 +192,8 @@ number
 {: .label .fs-1 }
 
 The remaining stock for the extra, if the extra has unlimited inventory this will return `nil`.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return the remaining stock for the extra on the given date.
+- When accessed independently or through a product, it will return the remaining stock for the extra on the next upcoming date (if there is one) or the most recent date.
```

Comment:
Should this now be updated based on this thread? https://easolhq.slack.com/archives/C03AJ5F6L1L/p1689348944793599